### PR TITLE
feat: remove `init` and `create-definition` command [EXT-2746]

### PR DESCRIPTION
**Pull request by andipaetzold**
## npx create-contentful-app create-definition

image

## npx create-contentful-app init

image

## npx create-contentful-app --help

image

### CODE REVIEW

**Review by kdamball — APPROVED** [review]
Maybe also a PR in doc app to go with it? https://www.contentful.com/developers/docs/extensibility/app-framework/create-contentful-app/

File: packages/contentful--create-contentful-app/lib/index.js [hunk 1]
```diff
@@ -61,6 +57,29 @@ async function promptAppName() {
 
 async function initProject(appName, options) {
   try {
+    if (appName === 'create-definition') {
```
[the remark this answers is not in the document]
Comment (andipaetzold) [reply packages/contentful--create-contentful-app/lib/index.js:60]: Only the check for `create-definition` can be extracted as `init` also sets `appName` to `undefined`. But I can extract to make it easier to read.

**Comment by andipaetzold** [comment]
> Maybe also update to docs go with it? https://www.contentful.com/developers/docs/extensibility/app-framework/create-contentful-app/

Will do. Good point!

**Review by colomolo — APPROVED** [review]
Looks good!

**Pull request merged**